kolibri/plugins/learn/assets/src/views/ChannelRenderer/CustomContentRenderer.vue

```diff
@@ -114,19 +132,33 @@
       // called in mainClient.js
       fetchContentCollection(message) {
         const { options } = message;
+        const { kinds, onlyContent, onlyTopics } = options;
+
+        if (onlyContent && onlyTopics) {
+          const err = new Error('onlyContent and onlyTopics can not be used at the same time');
+          return createReturnMsg({ message, err });
+        }
+
         return ContentNodeResource.fetchCollection({
           getParams: {
             ids: options.ids,
+            authors: options.authors,
+            tags: options.tags,
             parent: options.parent === 'self' ? this.topic.id : options.parent,
+            channel_id: this.topic.channel_id,
+            max_results: options.maxResults ? options.maxResults : 50,
+            cursor: options.cursor,
+            kind: onlyTopics ? ContentNodeKinds.TOPIC : undefined,
+            kind_in: onlyContent ? allButTopicTypes : kinds,
```

Comment:
I've discovered that this filter is already implemented in the `kind` filter, we just need to pass the value `content`. So I've updated the logic to use that filter instead of using the `kind_in` or create a new one.